genon/preprocessor/tests/unit/test_llm_cache_unit.py

```
@@ -0,0 +1,325 @@
+        return {"text": "ok2"}
+
+    out = m.cached_call("http://x", {"q": 4}, produce2)  # 손상 → miss → 재호출
+    assert calls["n"] == 1
+    assert out == {"text": "ok2"}
+
+
+def test_custom_serialize_deserialize_roundtrip(enabled_ctx):
+    def produce():
+        return ("content", {"tokens": 3}, "stop")
+
+    ser = lambda t: {"content": t[0], "usage": t[1], "finish": t[2]}
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `ser` as a `def`

(E731)